Framework/include/CommonVariables.h

```diff
@@ -84,13 +84,10 @@ class CommonVariables
         const auto& NGoodBJets_pt45 = tr.getVar<int>("NGoodBJets_pt45"+myVarSuffix_);
         const auto& GoodBJets_pt45  = tr.getVec<bool>("GoodBJets_pt45"+myVarSuffix_);
         
-        // Define Loose HEM15/16 veto
-        bool passHEMVetoLoose = !(objectInHEM(Muons,     -3.00, -1.30, -1.57, -0.87, 20.0, runYear) ||
-                                  objectInHEM(Electrons, -3.00, -1.30, -1.57, -0.87, 20.0, runYear));
-        tr.registerDerivedVar("passHEMVetoLoose"+myVarSuffix_, passHEMVetoLoose);
-
-        // Define full HEM15/16 veto
-        bool passHEMVeto = !objectInHEM(Jets,      -3.20, -1.10, -1.77, -0.67, 20.0, runYear) && passHEMVetoLoose; 
+        // Define HEM15/16 veto
+        bool passHEMVeto = !(objectInHEM(Jets,      -3.20, -1.10, -1.77, -0.67, 20.0, runYear) ||
+                             objectInHEM(Muons,     -3.00, -1.30, -1.57, -0.87, 20.0, runYear) || 
+                             objectInHEM(Electrons, -3.00, -1.30, -1.57, -0.87, 20.0, runYear));
```

Comment:
Why did you remove the passHEMVetoLoose. Were you the only person using it?

Reply:
Whoops not sure how that happened, I reset the file to the master version and committed it